DOCSP-1034a: fixed incorrect image

steveren opened the pull request.

steveren commented:
Staged at: https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/steverenaker/DOCSP-1034a/connect/tableau.html#connect-from-tableau-without-authentication-or-tls-ssl

atsansone commented on source/connect/tableau.txt:
```diff
@@ -31,7 +31,7 @@ listening, and click :guilabel:`Sign In` to connect.
 
 By default, :program:`mongosqld` binds to port ``3307``.
 
-.. figure:: /images/bi-connector/tableau.png
```
Believe it or not, the image needs to go here: https://github.com/mongodb/docs-assets/tree/bi-connector

atsansone replied:
Also, just caught that I messed up the *Server* field. It should probably say `localhost` like the others.

breinero commented:
LGTM

kay-kim commented:
@steveren / @atsansone  -- the images are kept within the bi-connector repo going forward.  An email about this was sent to the group in January.

kay-kim commented:
@steveren  -- if you could fix the second image as a separate commit to this branch, I can merge and publish.

atsansone commented:
@kay-kim : It did not work when I tried it a week ago. @i80and told me to put it in `docs-assets` and it worked. I'll circle back.

kay-kim commented:
It didn't work a week ago because after the email went out in January, no one actually took that step for their respective repo.  I noticed it while doing a merge on Monday, so I went ahead and made the change to this repo -- well, specifically, the master branch anyhow.

i80and commented:
Sorry for the wrong instructions, @atsansone. I should have remembered that we were moving away from docs-assets.

steveren commented:
OK, I believe all the images are now correct. I fixed the other one which still had MySQL in it and updated the one which had 127.0.0.1 instead of localhost.

kay-kim commented:
merged!
